### PR TITLE
Added several simple commands

### PR DESCRIPTION
**Pull request by naps250**
Added roll (1-100) by default and custom roll.
Added IQ and SR rolls.
Added global Random instance.

### CODE REVIEW

File: SteakBot.Core/ThreadLocalRandom.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /// </summary>
+    public static class ThreadLocalRandom
+    {
+        /// <summary>
+        /// Random number generator used to generate seeds,
+        /// which are then used to create new random number
+        /// generators on a per-thread basis.
+        /// </summary>
+        private static readonly Random globalRandom = new Random(Guid.NewGuid().GetHashCode());
+        private static readonly object globalLock = new object();
```
Comment (penev92) [inline SteakBot.Core/ThreadLocalRandom.cs]: Static readonly fields should use PascalCase naming.

File: SteakBot.Core/ThreadLocalRandom.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        /// <summary>
+        /// Random number generator
+        /// </summary>
+        private static readonly ThreadLocal<Random> threadRandom = new ThreadLocal<Random>(NewRandom);
```
Comment (penev92) [inline SteakBot.Core/ThreadLocalRandom.cs]: PascalCase please.

File: SteakBot.Core/Modules/RollModule.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading;
```
Comment (penev92) [inline SteakBot.Core/Modules/RollModule.cs]: First two `using` statements are unused.

**Review by penev92 — APPROVED** [review]

File: SteakBot.Core/Modules/RollModule.cs (hunk 3 above)
Comment (naps250) [reply SteakBot.Core/Modules/RollModule.cs]: Removed.

File: SteakBot.Core/ThreadLocalRandom.cs (hunk 2 above)
Comment (naps250) [reply SteakBot.Core/ThreadLocalRandom.cs]: Corrected.

File: SteakBot.Core/ThreadLocalRandom.cs (hunk 1 above)
Comment (naps250) [reply SteakBot.Core/ThreadLocalRandom.cs]: Corrected.

**Comment by penev92** [comment]
👍

**Pull request merged**